doc/reference/networking/net_mgmt.rst

```diff
@@ -47,7 +47,9 @@ Listening to network events
 
 You can receive notifications on network events by registering a
 callback function and specifying a set of events used to filter when
-your callback is invoked.
+your callback is invoked. The callback  will have to be unique for a
```

Comment:
I'm not quite sure what you're saying here.  What is a layer/code pair?  Does code mean an event code?  What is the "command part"?  Can you try rewriting this to explain this better?